packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -254,9 +254,8 @@ const makeParamManagerBuilder = zoe => {
     return param.getVisibleValue(proposed);
   };
 
-  const getParamList = () => {
-    return harden(namesToParams.keys().map(k => getParam(k)));
-  };
+  const getParamList = () =>
+    harden(mapIterable(namesToParams.keys(), k => getParam(k)));
```

Comment:
asking to learn:   does this change  the semantics, or it a safer way of invoking the same operation?  What's better about the new way?

Reply:
In this case, I chose to restore correctness not by flattening to an array, but rather by working with the iterable as an iterable. My `mapIterable` function does for iterables what `.map` does for arrays. A consequence here is that `getParamList` now returns an Iterable rather than an Array. 

In light of your question I notice that `ParamDescriptions` needed to be upgraded, so I'll now do that as part of this PR. I didn't notice that before because `yarn lint` didn't complain. It still doesn't complain after the fix. AFAICT, nothing actually calls `getParamList`.

Since it now returns an Iterable rather than an Array, would you prefer a name other than `getParamList`? Since it returns a `ParamDescriptions` should it be named `getParamDescriptions`?